src/main/groovy/nebula/plugin/dependencylock/DependencyLockTaskConfigurer.groovy

```diff
@@ -269,14 +269,17 @@ class DependencyLockTaskConfigurer {
                         if (ext != null) {
                             Collection<Configuration> lockableConfigurations = lockableConfigurations(project, subproject, ext.configurationNames, extension.skippedConfigurationNamesPrefixes)
                             Collection<Configuration> configurations = filterNonLockableConfigurationsAndProvideWarningsForGlobalLockSubproject(subproject, ext.configurationNames, lockableConfigurations)
-
+                            Configuration aggregate = subproject.configurations.create("aggregateConfiguration")
```

Comment:
This is probably the biggest change. I'm kind of creating custom "default" configuration. All resolvable configurations from a submodule are added to `aggregateConfiguration` which is created in each submodule and then all `aggregateConfiguration` are added to detached configuration where they are resolved together for global dependency lock

Reply:
Fancy! It does make the logic later simpler, too, with finding configurations that can be locked

Reply:
oh global locks :P this is great!